party/tribute/ohno-itshimagain.md

```diff
@@ -6,7 +6,41 @@
  * **Combat Role:** Damage + Defense
  * Abilities that sacrifice control and/or risk friendly fire.
 
+### Attributes and Skills
+
+ * **Maximum HP:** 65
+ * **STR:** +5
+ * **CON:** +4
+ * **DEX:** +1
+ * **INT:** -5
+ * **WIS:** -1
+ * **CHA:** +2
+
+### Defenses
+
+ * **REF:** 17
+ * **FORT:** 16
+ * **WILL:** 8
+
+### Abilities
+
+**Hit Them With the Pointy Bits** At-will, Melee, Standard. +3 vs. REF. On hit, deal 1d12 damage.
+
+**But That Looks Like It's Going to Hurt!** At-will, Standard. Gain a +3 to REF and FORT defenses, but suffer a -3 penalty to WILL, until the beginning of your next turn.
+
+**Can't Stop, Won't Stop** Charges: 2, Melee, Standard. +5 vs. REF. On hit, deal 1d12 damage and you *must* repeat this attack on your next turn as a Minor action against a different target if able.
+
+**Aaaaaaaaaaaaaah!** Charges: 2, Melee Burst, Standard. +3 vs. REF. Deal 1d12 damage to each target that is hit. If co-located with an ally, deal an additional 1d6 damage to all hit targets.
+
+**Ohno You Don't** Charges: 3, Melee, Minor. +3 vs. FORT. Prevent the target from using any move actions until the end of their next turn. On hit, also deal 1d6 damage.
+
+**Hurl** Charges: 2, Ranged, Standard. Must be co-located with an ally. +3 vs. REF. Throw your ally at a non-adjacent enemy target. Your ally is moved to the target's node and suffers 1d6 damage. On hit, the target suffers 2d12 damage.
```

Comment:
I'm proud of this one. Notice that, for maximum fun, your ally's damage is _not_ contingent on hit while your enemy's damage is.